src/Behat/Testwork/Tester/Exception/WrongPathsException.php

```diff
@@ -20,30 +20,30 @@
 final class WrongPathsException extends RuntimeException implements TesterException
 {
     /**
-     * @var string
+     * @var string[]
      */
-    private $path;
+    private $paths;
 
     /**
      * Initializes exception.
      *
-     * @param string $message
-     * @param string $path
+     * @param string   $message
+     * @param string[] $paths
      */
-    public function __construct($message, $path)
+    public function __construct($message, $paths)
     {
         parent::__construct($message);
 
-        $this->path = $path;
+        $this->paths = $paths;
     }
 
     /**
      * Returns path that caused exception.
      *
-     * @return string
+     * @return string[]
      */
-    public function getPath()
+    public function getPaths()
```

Comment:
Please also add `getPath()` with `@deprecated` tag for BC. It can return the first path given.